js/interface-directory.js

```diff
@@ -46,7 +46,7 @@ var DataDirectoryForm = (function() {
     }
 
     if (typeof this.directoryConfig.enable_thumbs === 'undefined' && this.directoryConfig.thumbnail_field !== '') {
-      this.directoryConfig.enable_thumbs = true;
+      this.directoryConfig.enable_thumbs = false;
```

Comment:
no thumbnails by default?

Reply:
Nope. When enabling thumbnails by default, the interface also asks for a folder. It's best to keep this disabled, and we can enable it for the specific layouts/app templates if necessary.